docs/embedded.adoc

```diff
@@ -519,6 +527,11 @@ sh $CHE_PROJECTS_ROOT/dg8-workshop/dg8-embedded-quarkus/scripts/load.sh
 
 Reload *PREVIEW* right away and you will see the entries again. But this time note that there are only two entries. And thats what we had specified in our Cache configuration.
 
+[NOTE]
+====
```

Comment:
Inn this note a couple of things are introduced, which is great, however we need to add more information for the users to understand the context.